### PR TITLE
SITE-1135 & PATIOS-263 import localizations

### PR DESCRIPTION
**Pull request by larkin**
Add base files for lt-lt and ka-GE.

### CODE REVIEW

File: ResearchKit/Localized/ka-GE.lproj/ResearchKit.strings [hunk 1]
```diff
@@ -0,0 +1,1479 @@
...
+"AX_DISMISS_KEYBOARD_CUSTOM_ACTION" = "კლავიატურაზე უარის თქმა";
+
+/* (No Comment) */
+"AX_GRAPH_AND_SEPARATOR" = " და ";
+
+/* (No Comment) */
+"AX_GRAPH_POINT_%@" = "წერტილი: %@";
+
+/* (No Comment) */
+"AX_GRAPH_RANGE_FORMAT_%@_%@" = "დიაპაზონი: % @-დან% @-მდე";
```
Comment (larkin) [inline ResearchKit/Localized/ka-GE.lproj/ResearchKit.strings]: broken format string

File: ResearchKit/Localized/ka-GE.lproj/ResearchKit.strings [hunk 2]
```diff
@@ -0,0 +1,1479 @@
...
+"AX_SIGNVIEW_LABEL" = "ხელმოწერის ველი";
+
+/* (No Comment) */
+"AX_SIGNVIEW_SIGNED" = "ხელმოწერილი";
+
+/* (No Comment) */
+"AX_SIGNVIEW_UNSIGNED" = "ხელმოუწერელი";
+
+/* (No Comment) */
+"AX_SLIDER_LABEL" = "პასუხის სლაიდერი. დიაპაზონი % @-დან %@-მდე";
```
Comment (larkin) [inline ResearchKit/Localized/ka-GE.lproj/ResearchKit.strings]: broken format string

**Comment by yuri-med** [comment]
> Add base files for lt-lt and ka-GE.

ka-GE is added already.

Should lt-LT have English strings inside?

File: ResearchKit/Localized/ka-GE.lproj/ResearchKit.strings (hunk 1 above)
Comment (yuri-med) [reply ResearchKit/Localized/ka-GE.lproj/ResearchKit.strings]: fixed

File: ResearchKit/Localized/ka-GE.lproj/ResearchKit.strings (hunk 2 above)
Comment (yuri-med) [reply ResearchKit/Localized/ka-GE.lproj/ResearchKit.strings]: fixed

**Comment by yuri-med** [comment]
@larkin ro-RO removed.

**Pull request merged**